appconfigservice/appconfig.yaml

```diff
@@ -131,6 +131,21 @@ paths:
           description: AppConfig not found
         500:
           description: Internal error
+  /app/configs/version:
+    get:
+      tags:
+        - App Configuration
+      summary: version endpoint
+      description: Public endpoint which returns the latest version of appconfig BB
+      responses:
+        200:
```

Comment:
It looks like by default the response content type is application/json. As per the issue description, we need to have content type text/plain for this end point. Please see https://swagger.io/docs/specification/describing-responses/ for reference and fix this. Thanks.